backend/src/ai_ticket_platform/routers/articles.py

```
@@ -0,0 +1,161 @@
+			"status": "success",
+			"message": f"Approval queued for article {article_id}",
+			"article_id": article_id,
+			"job_id": approval_job.id,
+			"intent_id": article.intent_id,
+			"version": article.version,
+		}
+
+	except HTTPException:
+		raise
+	except Exception as e:
+		logger.error(
```

Ruff (0.14.7)

Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)